psamm/datasource/sbml.py

```diff
@@ -798,6 +803,21 @@ def write_model(self, file, model):
         for rxn_id, lower_lim, upper_lim in model.parse_limits():
             flux_limits[rxn_id] = lower_lim, upper_lim
 
+        # Objective
+        objectivelist = ET.SubElement(model_tag, _tag(
+            'listOfObjectives', FBC_V2))
+        objectivelist.set(_tag('activeObjective', FBC_V2), 'O_1')
+        objective_tag = ET.SubElement(objectivelist, _tag('objective', FBC_V2))
+        objective_tag.set(_tag('id', FBC_V2), 'O_1')
+        objective_tag.set(_tag('type', FBC_V2), 'maximize')
+        fluxobjectivelist = ET.SubElement(objective_tag, _tag(
+            'listOfFluxObjectives', FBC_V2))
+        fluxobjective_tag = ET.SubElement(fluxobjectivelist, _tag(
+            'fluxObjective', FBC_V2))
+        fluxobjective_tag.set(_tag('reaction', FBC_V2), biomass_id)
+        fluxobjective_tag.set(_tag('coefficent', FBC_V2), '1')
+
+        gene_list = set()
```

Comment:
This belongs in the gene commit, I think? Also, probably a bit confusing calling this `gene_list` when it is a set.